### DISCUSSION TITLE
Getting error like : There is a problem with this repository on disk.

### DISCUSSION THREAD

**Discussion by sukumar27588** [community/community#54729] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

There is a problem with this repository on disk.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Hi @sukumar27588 , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction. Please note, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
